src/content/reference/manifest.mdx

```diff
@@ -610,6 +610,34 @@ volumes:
 
 Sets the working directory of your development container.
 
+### forward ([string], optional)
+
+Port forwards activated on any `okteto up` command, independently of the dev container name.
```

Comment:
What do we mean by this? I'm not sure what the "dev container name" refers to here.

Reply:
I mean that no matter which service is selected for `okteto up <svc>`, if the manifest sets a global forward, it will be used for any `okteto up` using that manifest.

Reply:
then let's say "global port forwards are always activated, independently of the service you are developing on". 

```suggestion
Global port forwards are always activated, independently of the service you are developing on.
```